resources/frontend-development.md

```diff
@@ -8,3 +8,11 @@ Here is a list of useful websites for learning and mastering CSS:
 - [freeCodeCamp](https://www.freecodecamp.org/learn/)
 - [Flexbox Frogy](https://flexboxfroggy.com/)
 - [CSS Grid Garden](https://cssgridgarden.com/)
+
+## JavaScript Library
+
+A javaScript library is a toolbox of pre-written code designed for a specific task. In other words,it's a collection of reusable functions and features framed or crafted to make certain programming tasks easier. For a beginner, think of it as a set of tools that you can use to build a web application without having to create everything from scratch. An example of a JavaScript library is React.
+
+Here is a useful YouTube video tutorial to learn and master React:
+
+- [freeCodeCamp](https://www.youtube.com/watch?v=bMknfKXIFA8&t=853s)
```

Comment:
I like the description. However, so far, we don't have description for other resources. We might want to have consistency here for the future of our website.

I would love to have thoughts from @mariahlaqua and @saragerion on this matter.

Here are two proposals:

**1. Without description**:

```suggestion
- [React Course - Beginner's Tutorial for React JavaScript Library](https://www.youtube.com/watch?v=bMknfKXIFA8&t=853s)
```

**2. With description**:

```suggestion
- **[React Course - Beginner's Tutorial for React JavaScript Library](https://www.youtube.com/watch?v=bMknfKXIFA8&t=853s)**:

   This is a React beginner's course by Bob Ziroll for freeCodeCamp. You will learn React by building eight real-world projects and solving 140+ coding challenges in this course.
```

Reply:
I will wait for the opinion of the other maintainers. Thank you, @adiati98, for the review.

Reply:
Indeed we do not have a consistent practice with adding descriptions, and it is not stated as a requirement in the issue. I would say that we approve and merge the original pull request.

Then I would create a separate issue to improve the quality of resources by adding descriptions, and optionally screenshots or gifs. Once that is done we can update the original issue to include writing a description.

That's my thoughts @RitaDee @adiati98 @saragerion

Reply:
Noted, @mariahlaqua. So, as we want to add description to resources moving forward — which can benefits readers to get a brief info about the resource — we'll include a description here. 👍

@RitaDee can you please make changes and go with suggestion no. 2? Thanks! 😊

Reply:
Agree with @mariahlaqua :)

Reply:
I've made a new issue, https://github.com/shesharpnl/knowledge-hub/issues/53

regarding adding descriptions.

Reply:
Thank you for the review, @mariahlaqua, @saragerion and @adiati98. I will make the changes you suggested.